downloads-generation/models_class1_allele_specific_single_kim2014_only/models.py

```diff
@@ -0,0 +1,16 @@
+import sys
+from mhcflurry.class1_allele_specific.train import HYPERPARAMETER_DEFAULTS
+import json
+
+models = HYPERPARAMETER_DEFAULTS.models_grid(
+    #impute=[False, True],
+    impute=[False],
```

Comment:
Not using imputation because of the failure we saw previous for alleles without any data at all?